src/features/devtools/DevToolsDrawer.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import {
+  Checkbox,
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerHeader,
+  FormControl,
+  FormLabel,
+  Radio,
+  RadioGroup,
+  Stack,
+  Wrap,
+  useColorMode,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { useTranslation } from 'react-i18next';
+
+import { AVAILABLE_LANGUAGES } from '@/lib/i18n/constants';
+
+export const DevToolsDrawer = ({
+  disclosure,
+}: {
+  disclosure: ReturnType<typeof useDisclosure>;
+}) => {
+  const { colorMode, setColorMode } = useColorMode();
+
+  const { t, i18n } = useTranslation(['common']);
+
+  return (
+    <Drawer
+      onClose={disclosure.onClose}
+      isOpen={disclosure.isOpen}
+      size="xs"
+      placement="left"
+    >
+      <DrawerContent>
+        <DrawerCloseButton />
+        <DrawerHeader>Development panel</DrawerHeader>
+        <DrawerBody>
+          <Stack spacing={6}>
+            <Checkbox
+              isChecked={colorMode === 'dark'}
+              onChange={(e) =>
+                setColorMode(e.target.checked ? 'dark' : 'light')
+              }
+            >
+              Dark mode
+            </Checkbox>
+            <FormControl>
+              <FormLabel>Language</FormLabel>
+              <RadioGroup
+                value={i18n.language}
+                onChange={(newValue) => i18n.changeLanguage(newValue)}
+              >
+                <Wrap spacing={4}>
+                  {AVAILABLE_LANGUAGES.map((language) => (
+                    <Radio key={language.key} value={language.key}>
+                      {t(`common:languages.${language.key}`)}
+                    </Radio>
+                  ))}
+                </Wrap>
+              </RadioGroup>
+            </FormControl>
```

Comment:
IMO we can extract these components into external files; we can see them as a lib of standalone dev components we can add / remove based on project needs :) And I think it will make the drawer more readable in long term, even if it's only development tools

I don't know if it is too much or not, but I prefer to see something like this:

```tsx
<DrawerBody>
  <Stack spacing={6}>
    <DarkModeDevTool />
    <LanguageSwitcherDevTool />
  </Stack>
</DrawerBody>
```

instead of adding everything inside the drawer; So if I add a new tool for managing tanstack query (toggle dev tools, delete cache, etc...) I can just add <TansStackQueryDevTool /> and It Just Works™ :p 

I also know that I can update only DarkModeDevTool instead of finding related code inside the drawer for future updates :) 

I think you get the idea :p